sys-utils/unshare.c

```diff
@@ -990,16 +1001,24 @@ int main(int argc, char *argv[])
 			unshare_flags |= CLONE_NEWUSER;
 			if (!strcmp(optarg, "auto"))
 				insert_map_range(&groupmap,
-					read_subid_range(_PATH_SUBGID, real_euid));
+						 read_subid_range(_PATH_SUBGID, real_euid, 0));
+			else if (!strcmp(optarg, "subids"))
+				insert_map_range(&usermap,
+						 read_subid_range(_PATH_SUBGID, real_euid, 1));
```

Comment:
@dgibson coverity-scan reports this "&usermap" as a possible copy & past bug, and it really seems that we want "&groupmap" here. It's OPT_MAPGROUPS code block (--map-groups).  Can you please check the code?

Reply:
Yes, it should be `groupmap`.  Embarrasing.  Interestingly it's correct for `--map-subids` but not for `--map-groups=subids`.

You want me to send a fixup patch? Or will you handle it?

Reply:
I'll fix it. Thanks.